07-routing-gossip.md

```diff
@@ -382,6 +384,19 @@ individual bits:
 | 0             | `direction` | Direction this update refers to. |
 | 1             | `disable`   | Disable the channel.             |
 
+The `message_flags` bitfield is used to indicate the presence of optional
+fields in the `channel_update` message:
+
+| Bit Position  | Name                      | Field                            |
+| ------------- | ------------------------- | -------------------------------- |
+| 0             | `option_channel_htlc_max` | `htlc_maximum_msat`              |
```

Comment:
I'd rather have this in the `channel_announcement`, maybe we can use the features bitfield to signal this? Anyway I expect everybody will just add this in perpetuity so a bit is not really needed.